company/views.py

```diff
@@ -251,26 +268,148 @@ def get_context_data(self, 	**kwargs):
 		context['upc'] = UserPostConnection.objects.filter(company_id= kwargs['pk'],status="Rejected") 
 		return context
 
-def AddQuestionView(request):
-		
+class GenerateQuestionView(TemplateView):
+	template_name = 'company/test.html'
+
+	def get_context_data(self, 	**kwargs):
+		context = super().get_context_data(**kwargs)
+		context['technology'] = Technology.objects.all()
+		return context 
+
+	def post(self, request, *args, **kwargs):
+		return HttpResponseRedirect('/company/addquestion/')
+
+class ListofQuestionView(TemplateView):
+	template_name = 'company/addquestion.html'
+
+	def get_context_data(self, 	**kwargs):
+		context = super().get_context_data(**kwargs)
+		context['technology'] = Technology.objects.all()
+		tech = Technology.objects.get(technology_name = kwargs['type'])
+		context['type'] = kwargs['type']
+		context['questions'] = Question.objects.filter(technology_id = tech.pk)
+		context['answers'] = Answers_HR.objects.all()
+		return context
 
+	def post(self,request, *args, **kwargs):
+		try:
+			question = request.POST.get('question')
+			answer = request.POST.get('answer')
+			print("before IF")
+			if request.POST.get('question') != "" and request.POST.get('answer') != "":
+
+				tech = Technology.objects.get(technology_name = kwargs['type'])
+				print("TEchnology is",tech)
+				obj1 = Answers_HR()
```

Comment:
give proper names, Answeres_HR should be AnsweresHR, obj1 is not proper, it should be answere_hr